README.md

```diff
@@ -3,7 +3,7 @@ Ethereum Classic RFC
 
 This is the Ethereum Classic RFC project.
 
-We collect specifications for APIs, file formats, protocols, processes and compositions of RFCs which result in a blockchain hard/soft fork.
```

Comment:
Softfork is where the community go with it, harkfork is where they don't. This process is desiigned to get everyone to reach *rough* consensus through executing code. So we aim for softforks and not hardforks.